`make styles-check` return error if you have untrack files

## Data
- **Shiori version**: last master branch
- **Database Engine**: sqlite
- **Operating system**: linux
- **CLI/Web interface/Web Extension**: 

## Describe the bug / actual behavior
if you have files that not add to git to track  `make style-check` return error
```
fatal: No names found, cannot describe anything.
CLEANCSS_OPTS=--with-rebase /usr/bin/bash ./scripts/styles_check.sh
❌ git tree is not clean. Please commit all changes before running this script.
make: *** [Makefile:85: styles-check] Error 1
```

## Expected behavior
styles-check  return error if track files not commit.

## To Reproduce
Steps to reproduce the behavior:
1. Clone master project 
2. run `make style-check`
3. everything work fine
4. create an empty file in root of project test.sh
5. run `make style-check` again
6. you see that error



## Notes
we have some files in project that not ready to add yet or some other is personal script that we use for easier development processes things that we not add to project or `.gitignore` so in this situation we can't use `make style-check` command.

--- a/scripts/styles_check.sh
+++ b/scripts/styles_check.sh
@@ -3,15 +3,15 @@
 # This script is used to check if the style.css file is up to date.
 
 # Check if the git tree for CWD is clean
-if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/css --porcelain)" ]; then
     echo "❌ git tree is not clean. Please commit all changes before running this script."
     exit 1
 fi
 
 # Check style.css file
 CLEANCSS_OPTS=${CLEANCSS_OPTS} make styles
-if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/css --porcelain)" ]; then
     echo "❌ style.css wasn't built from less changes. Please run 'make styles' and commit the changes."
-    git reset --hard
+    git checkout -- internal/view/assets/css/
     exit 1
 fi

--- a/scripts/styles_check.sh
+++ b/scripts/styles_check.sh
@@ -3,14 +3,14 @@
 # This script is used to check if the style.css file is up to date.
 
 # Check if the git tree for CWD is clean
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
     echo "❌ git tree is not clean. Please commit all changes before running this script."
     exit 1
 fi
 
 # Check style.css file
 CLEANCSS_OPTS=${CLEANCSS_OPTS} make styles
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
     echo "❌ style.css wasn't built from less changes. Please run 'make styles' and commit the changes."
     git reset --hard
     exit 1

--- a/scripts/swagger_check.sh
+++ b/scripts/swagger_check.sh
@@ -10,7 +10,7 @@ if [ "$CURRENT_SWAG_VERSION" != "$REQUIRED_SWAG_VERSION" ]; then
 fi
 
 # Check if the git tree for CWD is clean
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status docs/swagger --porcelain)" ]; then
     echo "❌ git tree is not clean. Please commit all changes before running this script."
     git diff
     exit 1
@@ -18,18 +18,18 @@ fi
 
 # Check swag comments
 make swag-fmt
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status docs/swagger --porcelain)" ]; then
     echo "❌ swag comments are not formatted. Please run 'make swag-fmt' and commit the changes."
     git diff
-    git reset --hard
+    git checkout -- docs/swagger
     exit 1
 fi
 
 # Check swagger documentation
 make swagger
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status docs/swagger --porcelain)" ]; then
     echo "❌ swagger documentation not updated, please run 'make swagger' and commit the changes."
     git diff
-    git reset --hard
+    git checkout -- docs/swagger
     exit 1
 fi

